docs/Developers/Newcomers_Guide.rst

```diff
@@ -333,6 +333,12 @@ and clicking ``New Pull Request``.
     updated, you can edit your comment instead. If you need to add new
     information, make a new comment.
 
+    If you are generating a pull request for a particular issue, you need to
```

Comment:
I think generating is not the most appropriate word that can be used here. Perhaps you could write something like "If you are making.."